ffmpeg_patches/cbs/avconfig.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/libavutil/avconfig.h b/libavutil/avconfig.h
```

Comment:
If you have any idea the purpose or original source of these patches, can you add some details to the top like in the other patches?

(Applies to all 4 patch files)